resources/qr_code.go

```diff
@@ -0,0 +1,45 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/razorpay/razorpay-go/constants"
+	"github.com/razorpay/razorpay-go/requests"
+)
+
+//QrCode ...
+type QrCode struct {
+	Request *requests.Request
+}
+
+// Create creates a new qrcode for the given data.
+func (q *QrCode) Create(data map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
+	return q.Request.Post(constants.QRCODE_URL, data, extraHeaders)
+}
+
+//Fetch fetches the qrcode entity having the given QrCodeID.
+func (q *QrCode) FetchPayments(QrCodeID string, queryParams map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
+	url := fmt.Sprintf("%s/%s/payments", constants.QRCODE_URL, QrCodeID)
```

Comment:
While building the url, the QrCodeId should be url escaped as the input Id can be passed with a `/` and can indicate a different url.


Please check other urls as well that are being built via `fmt.Sprintf`

Reply:
added escape in all get methods api's